cypress/integration/testBorna/test.cy.js

```diff
@@ -0,0 +1,179 @@
+import uniqid from "uniqid";
+
+function getPlaceholder(placeholder) {
+    return cy.get('input[placeholder="' + placeholder + '"]');
+}
+
+function makeid(length) {
+    var result = "";
+    var characters = "abcdefghijklmnopqrstuvwxyz";
+    var charactersLength = characters.length;
+
+    for (var i = 0; i < length; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+}
+
+context("Blocks Page", () => {
+    var import_url =
+        "https://wby-fm-bucket-d7ecf9a.s3.amazonaws.com/EXPORTS/8lktvc8xe-WEBINY_BLOCK_EXPORT.zip?AWSAccessKeyId=ASIA4ITYNEHPU62DZ7E2&Expires=1691594051&Signature=sTOI9ECU8yHy%2F7NLf%2Bx52h%2FQa8A%3D&X-Amzn-Trace-Id=Root%3D1-64ca72bf-24750f034e3441390ab83f1a%3BParent%3D2def68bf20ca0ba7%3BSampled%3D0%3BLineage%3D8b7b8b32%3A0%7Cc2636230%3A1%7C2391eed0%3A0&x-amz-security-token=IQoJb3JpZ2luX2VjEFcaCXVzLWVhc3QtMSJHMEUCIElvm8HhZTRC09kHKjn6cHCD%2BZu3HoZYDst22zZTVeDVAiEAuCo42IE3VAvxUDSOO%2FtXmG63aK8AgQMndKpDQ5KXtfcqmAMI8P%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARAAGgw4NDMxMzk4NTA3MTkiDCDmvtpd8IGv212iBirsAmfisv1Fl%2BuJumaxFVCVBkMSVECa%2BTeYPREA50PvGgl7v4F%2FKVmJpR4kXkTTehNp40%2Bxqa%2F%2BXV2ulON7LQjkt1RpsokJ%2F9fjmP5oxWAc8%2Fv0twL3texB%2FBkq2hz%2Bw6D5Ay3BS9Hf92MiOektY2TuNJFCxUpHmQPI1EvLBQ7BqsRJYKHTFnUHdPQlIjzjG8QKOwO24kinGx6HxATfMnUn5Nu5qdDO3C8JErFC0LAAJyJFJu0bYFGSGf4QDJzE9xXtzfoeBHPMnLpQeALalOivg7g3QmL9wx%2BcWqQsYzuEEYkffQsIPFbY607pUDxcGQvM7gT91cKuXo4KNXkkbHzxE2OqJSHdQ8RQMRa0REtl5uCHsfN3%2FfjOh1FT0tjXfybCVrPxK6zar3fqMKEsZdRZ9x4bVoQp8ti2HPdbD0IxUtKWOZpPBMkdTzgV442zYk%2BrdcJ5opUpAvE10zdMbrqY8IIfHECQLMbjjfm5ZcIwsdeppgY6nQEW2k5HstUcrtwEJxfjUW1YP1mitNyPjwjhhw0MJxEmsMnEnt%2F8Ci9SRZ51qtXdUx%2F%2BJgvLBe%2FzQDKFYEAiZZbQfm9MBtNBpHKZRttNKRua1YbychTatZfTnIiaT0Pb%2F6jNJ0FHafQTYQpx9Z4a6nXeCN%2FJwSdoaSTbwpfSs2oJ5kpcaZK5aonMgkSvLJ9x6vMJMCOGDpHHHQkljU1M";
+    var block_category_name = uniqid();
```

Comment:
When naming variables, we use `camelCase`. 

For consistency, let's rename all variables. For example, `block_category_name` to `blockCategoryName`.